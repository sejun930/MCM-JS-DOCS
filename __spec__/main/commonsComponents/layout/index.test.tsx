import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";

import LayoutPage from "src/main/commonsComponents/layout";

export default describe("Layout Home Page", () => {
  // 스냅샷
  test("Snapshot", () => {
    const { container } = render(
      <RecoilRoot>
        <LayoutPage>
          <div>레아이웃 페이지입니다.</div>
        </LayoutPage>
      </RecoilRoot>
    );
    expect(container).toMatchSnapshot();
  });
});
