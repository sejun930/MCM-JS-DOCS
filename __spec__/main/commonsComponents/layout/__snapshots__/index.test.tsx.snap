// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Home Page Snapshot 1`] = `
<div>
  <div
    class="layout-home-wrapper css-1twbe3w"
  >
    <header
      class="layout-header-wrapper css-9a7bum"
    >
      <a
        class="mcm-link-unit layout-header-link css-1h5x3dy"
        href="/"
      >
        <img
          class="mcm-image-unit layout-header-logo css-msby2z"
          src="/images/commons/logo/MCM_white_logo.png"
        />
      </a>
    </header>
    <div
      class="css-1ph01pu"
    >
      <nav
        class="nav-wrapper css-qfzuzk"
      >
        <article
          class="nav-list-wrapper css-jh0u6n"
        >
          <div
            class="css-peiozq"
          >
            <div
              class="nav-search-wrapper css-1pry9ao"
            >
              xw
              <fieldset
                class="css-1opjr3t"
              >
                <legend>
                  모듈 검색
                </legend>
                <div
                  class="css-1rbly99"
                >
                  <div
                    class="css-2gjeij"
                  >
                    <input
                      class="nav-search-input css-1lp61su"
                      maxlength="20"
                      placeholder="모듈 검색"
                      type="text"
                    />
                    <div
                      class="css-iju4mq"
                    >
                      <button
                        class="mcm-close-button-unit css-1yn8fww"
                      />
                    </div>
                  </div>
                </div>
              </fieldset>
            </div>
          </div>
          <ul
            class="nav-list-wrapper css-2yaic1"
          >
            <li>
              <a
                class="mcm-link-unit css-1h5x3dy"
                href="/modules/Modal"
              >
                <span
                  class="mcm-span-unit css-1oteowz"
                >
                  Modal
                </span>
              </a>
            </li>
            <li>
              <a
                class="mcm-link-unit css-1h5x3dy"
                href="/modules/Slick"
              >
                <span
                  class="mcm-span-unit css-1oteowz"
                >
                  Slick
                </span>
              </a>
            </li>
          </ul>
        </article>
      </nav>
      <div>
        레아이웃 페이지입니다.
      </div>
    </div>
  </div>
</div>
`;
